client-src/elements/chromedash-userlist.ts

```diff
@@ -75,14 +71,17 @@ class ChromedashUserlist extends LitElement {
   }
 
   _onAdminToggle() {
-    const formEl = this.shadowRoot.querySelector('form');
-    const adminCheckbox = formEl.querySelector('input[name="is_admin"]');
-    const siteEditorCheckbox = formEl.querySelector(
+    const formEl: HTMLFormElement | null =
+      this.renderRoot.querySelector('form');
+    const adminCheckbox = formEl?.querySelector(
+      'input[name="is_admin"]'
+    ) as SlCheckbox;
```

Comment:
Are you sure `SlCheckbox` is the right type for an `<input>` element?

Reply:
it seems like SlCheckbox is used throughout similar use cases and also here in https://github.com/GoogleChrome/chromium-dashboard/blob/3f8901bc4cf7cb2dd841066f120389ed6ef76e8e/client-src/elements/chromedash-form-field.ts#L264

Reply:
Yeah, when you see `<sl-checkbox>`, it'll have the `SlCheckbox` type, but when you see `<input>` it'll be `HTMLInputElement`.

Reply:
This one should be `as HTMLInputElement`.

You can see on line 151 that this element is just an old fashioned `<input>` element.   I believe that this form was created before we started using Shoelace and we never upgraded it to use Shoelace.